pkg/store/multitsdb.go

```diff
@@ -178,12 +183,12 @@ func (s *MultiTSDBStore) Series(r *storepb.SeriesRequest, srv storepb.Store_Seri
 
 		for tenant, store := range stores {
 			store := store
-			seriesCtx, closeSeries := context.WithCancel(gctx)
+			seriesCtx, cancelSeries := context.WithCancel(gctx)
 			seriesCtx = grpc_opentracing.ClientAddContextTags(seriesCtx, opentracing.Tags{
 				"tenant": tenant,
 			})
-			defer closeSeries()
-			ss := newSeriesSetServer(seriesCtx, respSender)
+			defer cancelSeries()
+			ss := newTenantSeriesSetServer(seriesCtx, tenant, respSender)
```

Comment:
What about `warnings` on series? are those wrapped with prefix?

Reply:
yeah it was already the case, but slightly inconsistent (just generically wrapped the error and used the same in warnings and error handling, but that's not how we prefix other warnings in other places, so will do that instead)

Reply:
done